src/wazuh_modules/inventory_harvester/src/systemInventory/systemContext.hpp

```diff
@@ -1079,6 +1081,30 @@ struct SystemContext final
         return installTimeRaw;
     }
 
+    std::string_view hotfixName()
+    {
+        if (m_type == VariantType::Delta)
+        {
+            if (m_delta->data_as_dbsync_hotfixes() && m_delta->data_as_dbsync_hotfixes()->hotfix())
+            {
+                return m_delta->data_as_dbsync_hotfixes()->hotfix()->string_view();
+            }
+        }
+        else if (m_type == VariantType::SyncMsg)
+        {
+            if (m_syncMsg->data_as_state() && m_syncMsg->data_as_state()->attributes_as_syscollector_hotfixes() &&
+                m_syncMsg->data_as_state()->attributes_as_syscollector_hotfixes()->hotfix())
+            {
+                return m_syncMsg->data_as_state()->attributes_as_syscollector_hotfixes()->hotfix()->string_view();
+            }
+        }
+        else
+        {
+            return "";
```

Comment:
We are missing:
- The JSON case
- The changes at `wdb.c`
- The changes at `wdb_integrity.c`

Reply:
I think the required changes in wdb.c and wdb_integrity.c are already implemented, but the JSON case was really important for deletion. Thanks!

Reply:
> I think the required changes in wdb.c and wdb_integrity.c are already implemented,

You are right!